_examples/workers/README.MD

```diff
@@ -0,0 +1,173 @@
+# Javascript Workers
```

Comment:
Este arquivo precisa ter os links para voltar para o `README.md`, como link para o início, e outro com o link para `examples.md`, como link para voltar;
**Exemplo:**
```markdown
# Exemplos
[Início](../../../README.md) |
[Exemplos](../../../examples.md)
...
```

Reply:
Atualizei

Reply:
Só faltou referenciar o exemplo no examples.md e colocar uns links no .md do seu exemplo.

Reply:
Só faltou referenciar o exemplo no `examples.md`